### ISSUE TITLE
HTML entities

### ISSUE THREAD

**Issue by josephzidell** [intuit/QuickBooks-V3-PHP-SDK#41]
We encountered an issue in `Php2Xml.php` when it creates a DOM element [here](https://github.com/intuit/QuickBooks-V3-PHP-SDK/blob/master/src/XSD2PHP/src/com/mikebevz/xsd2php/Php2Xml.php#L314) with HTML chars. Fix we did:

Old:
```php
$el = $this->dom->createElement($code.":".$propDocs['xmlName'], $value);
```

Fix:
```php
$el = $this->dom->createElement($code.":".$propDocs['xmlName'], htmlspecialchars($value));
```

**Comment by hlu2** [comment]
@josephzidell 
I will include it in the code. Thanks for pointing it out :)

**Comment by josephzidell** [comment]
Awesome! Thanks

**Comment by hlu2** [comment]
@josephzidell 
Fixed on v3.3.0, thanks letting me know, joseph! :)
